redux/actions/contactUs.js

```diff
@@ -0,0 +1,20 @@
+import 'common/initialiseFirebase.js'
+import firebase from 'firebase'
+import moment from 'moment'
+
+export const handleNameChange = (inputValue) => {
+  return {type: 'contactUs/HANDLE_NAME_CHANGE', payload: inputValue}
```

Comment:
remove handle everywhere. All actions 'handle' something.

change them all like this `handleNameChange` -> `updateName`